app/models/draft.rb

```diff
@@ -9,8 +9,9 @@ class Draft < ActiveRecord::Base
 
   class OutOfSequence < StandardError; end
 
-  def self.set(user, key, sequence, data, owner = nil)
+  def self.set(user, key, sequence, data, owner = nil, force_save = false)
```

Comment:
I have the same feedback as in the JS world - this would be better served by a ruby keyword argument.